data/css/escola.scss

```diff
@@ -122,16 +132,38 @@ $display-font: 'Raleway Black' !default;
       }
 
       &__context {
-        color: #b3b3b3;
+        color: $card-context-color;
+        font-weight: normal;
+      }
+
+      &__title,
+      &__context {
+        padding-left: 10px;
       }
 
       padding-right: 20px;
       box-shadow: none;
-      margin: 0px 0px 2px 0px;
+      margin: 0px 100px 10px 100px;
     }
   }
 }
 
+.ContentGroupNoResultsMessage {
+  margin-top: 80px;
+  padding: 0;
+  &__title,
+  &__subtitle {
+    font-size: 1.25em;
+    color: white;
+    padding: 0em 1.25em 0em 1.25em;
+    font-style: normal;
+  }
+  &__subtitle {
+    margin-top: 1.0em;
+    padding: 0;
+  }
+}
```

Comment:
I think there is something off in composite mode with the no results page:

![virtualbox_endlessos_23_02_2017_16_21_56](https://cloud.githubusercontent.com/assets/481418/23275903/00a1bd40-f9e7-11e6-9114-fa39b5aa8233.png)